Problems with build:css on Windows

I was trying to build on Windows (Windows 10 Home, x64) but ran into an issue with the `build:css` script. When I run it I get the following error:
```bash
$ npm run build:css

> firefox-send@1.1.0 build:css D:\Workspace\send
> postcss 'frontend/src/*.css' -d public

× Input Error: You must pass a valid list of files to parse
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! firefox-send@1.1.0 build:css: `postcss 'frontend/src/*.css' -d public`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the firefox-send@1.1.0 build:css script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

When I changed the script to the following, I no longer had any problems:
```json
"scripts": {
  "build:css-current": "postcss 'frontend/src/*.css' -d public",
  "build:css-works-for-me": "postcss frontend/src/*.css -d public"
}
```

I would guess this is an issue with postcss. But I couldn't find an issue about it in their repository, which I would expect given its popularity, so I decided to create an issue here first. Anyway, if the script I provided above doesn't break the build process on other system, it could be change I guess?

--- a/package.json
+++ b/package.json
@@ -99,7 +99,7 @@
   ],
   "scripts": {
     "build": "npm-run-all build:*",
-    "build:css": "postcss 'frontend/src/*.css' -d public",
+    "build:css": "postcss frontend/src/*.css -d public",
     "build:js": "webpack -p",
     "build:vendor": "cp node_modules/l20n/dist/web/l20n.min.js node_modules/babel-polyfill/dist/polyfill.min.js public",
     "build:version": "node scripts/version",

